requirements.txt

```diff
@@ -18,6 +18,7 @@ pytest
 requests>=2.22.0
 s3fs==0.4.2
 scanpy
+tenacity
```

Comment:
Using [Tenacity](https://tenacity.readthedocs.io/en/latest/) instead of [retrying](https://pypi.org/project/retrying/) since the latter was last updated in 2014.